test/sanitizer_suppressions/lsan

```diff
@@ -1,7 +1,4 @@
 # Suppress warnings triggered in dependencies
-leak:libqminimal
-leak:libQt5Core
-leak:libQt5Gui
```

Comment:
23964: 
> With Qt 5.15.2 in depends

it could be too early for this, we still use 5.12.11

Reply:
dropped, please re-review